### DISCUSSION TITLE
Plugins installing Problem

### DISCUSSION THREAD

**Discussion by rieders** [inventree/InvenTree#5406] [General]
Hello
I installed a VM with Debian 11 on my Proxmox server.
I installed it using https://docs.inventree.org/en/latest/start/installer/inventree.
It's going great so far. Only plugins cannot be installed so easily or not at all.
I somehow managed to install the Brother printer.
Unfortunately I can't find any help in the manual. Why is it so difficult to install plugins?
I also tried it with Ubuntu 20.04. The result is almost the same.
Strangely enough, 3 plugins are activated there.

Maybe someone has had similar problems or a solution.

Otherwise I think inventorree is great.

Greetings
Andre

**Comment by wolflu05** [comment]
Hello, can you please share a bit more info about why you think it is difficult? 

Those 3 plugins you see are for sure builtin plugins handling qrcode scanning and pdf printing.

**Comment by rieders** [comment]
Now that I've tried to install inventree-supplier-panel, I only get the Apache default page after the restart.
Somehow inventree suddenly doesn't start anymore.

Greetings

**Reply by wolflu05** [reply]
That's not much info with what I can help you. If you want to solve the issue, you need to provide more info: how did you attempt install, logs, errors, ... otherwise we can't help you.
